h2/events.py

```diff
@@ -356,22 +356,50 @@ def __repr__(self):
         )
 
 
-class PingAcknowledged(Event):
+class PingReceived(Event):
     """
-    The PingAcknowledged event is fired whenever a user-emitted PING is
-    acknowledged. This contains the data in the ACK'ed PING, allowing the
+    The PingReceived event is fired whenever a PING is received. It contains
+    the 'opaque data' of the PING frame.
+
+    .. versionadded:: 3.1.0
+    """
+    def __init__(self):
+        #: The data included on the ping.
+        self.ping_data = None
+
+    def __repr__(self):
+        return "<PingReceived ping_data:%s>" % (
+            _bytes_representation(self.ping_data),
+        )
+
+
+class PingAckReceived(Event):
+    """
+    The PingAckReceived event is fired whenever a PING acknowledgment is
+    received. It contains the 'opaque data' of the PING+ACK frame, allowing the
     user to correlate PINGs and calculate RTT.
+
+    .. versionadded:: 3.1.0
     """
     def __init__(self):
         #: The data included on the ping.
         self.ping_data = None
 
     def __repr__(self):
-        return "<PingAcknowledged ping_data:%s>" % (
+        return "<PingAckReceived ping_data:%s>" % (
             _bytes_representation(self.ping_data),
         )
 
 
+class PingAcknowledged(PingAckReceived):
+    """
+    PingAcknowledged: same as PingAckReceived
+
+    .. deprecated:: 3.1.0
```

Comment:
I didn't because of f0dd593bf6eebdf5b81822edbd7a2e99af242ce0, and I can't remember hyper-h2 having used them for previous deprecated things.
But if you want one, I won't block it.